README.md

```diff
@@ -177,8 +174,9 @@ and you can do use your developer tools to debug.
 ### `happo upload_diffs`
 
 Uploads all current diff images to an Amazon S3 account and reports back URLs
-to access those diff images. Requires the `s3_access_key_id`,
-`s3_secret_access_key`, and `s3_bucket_name` configuration options.
+to access those diff images. Requires that `S3_ACCESS_KEY_ID`,
+`S3_SECRET_ACCESS_KEY`, `S3_BUCKET_NAME`, and `S3_BUCKET_PATH` are specified
+as environment variables.
```

Comment:
This isn't quite accurate since S3_BUCKET_PATH is not required.

It would be good to add some information about what each of these environment variables does and also show some code that demonstrates how to set them.